docs/services/navigation/_index.md

```diff
@@ -61,40 +67,40 @@ Click **Create service**:
         "store": {
             "type": "mongodb",
             "config": {
-                "uri": "mongodb://127.0.0.1:27017"
+                "uri": "mongodb://127.0.0.1:12345"
             }
         }
-        },
-        "movement_sensor": "test_movement",
-        "base": "test_base",
-        "obstacles":[{
-            "geometries": [{
-                "label":"aLabel",
-                "orientation":{
-                    "type":"ov_degrees",
+    },
+    "movement_sensor": "your-movement-sensor",
+    "base": "your-base",
+    "obstacles":[{
+        "geometries": [{
+            "label":"your-label-for-this-obstacle",
+            "orientation":{
+                "type":"ov_degrees",
                     "value":{
                         "x":1,
                         "y":0,
                         "z":0,
                         "th": 90
                     }
-                },
-                "x":10,
-                "y":10,
-                "z":10,
-                "translation":{
-                    "x":1,
-                    "y":1,
-                    "z":1
-                }
-            }],
-            "location":{
-                "latitude":1,
-                "longitude":1
+            },
+            "x":10,
+            "y":10,
+            "z":10,
+            "translation":{
```

Comment:
'translation' is not supported for obstacles.

Reply:
Addressed